lib/setup_nfs_functions.sh

```diff
@@ -97,7 +97,7 @@ recreateVolumeIfMountFromDifferentFolder() {
   local volume_path=$2
   local volume_type=$3
   local volume_opts=$4
-  if [[ $(docker volume inspect --format '{{.Options.device}}' ${volume_name}) != ":${volume_path}" ]]; then
+  if [[ $(docker volume inspect --format '{{.Options.device}}' ${volume_name} | sed 's/://g') != "${volume_path}" ]]; then
```

Comment:
If we are specifically looking for a single colon at the beginning of the string, we should anchor it there and remove the greedy flag.

```suggestion
  if [[ $(docker volume inspect --format '{{.Options.device}}' ${volume_name} | sed 's/^://') != "${volume_path}" ]]; then
```